### ISSUE TITLE
Internal Error: runtime.lua:634: attempt to index a nil value (global 'fieldName')

### ISSUE THREAD

**Issue by jbmorley** [inseven/opolua#346]
## Description

I found a database bug!

https://archive.org/download/3-libjune-05/3LIBJUNE05.iso/library%2Fepocutil%2Fepocdos%2Fepocdos.sis

## Details

```
runtime.lua:634: attempt to index a nil value (global 'fieldName')
EPOCDOS\CREATEUIDFILE:00015BBA: A5 [Create] logName=0 fields=A$
EPOCDOS\DOSCOREENGINEOS:00000338: 53 [RunProcedure] 0x0161 (name="CREATEUIDFILE" nargs=0)
stack traceback:
	runtime.lua:634: in method 'openDb'
	ops.lua:1458: in function 'ops.Create'
	runtime.lua:864: in function <runtime.lua:849>
	[C]: in function 'xpcall'
	runtime.lua:983: in method 'pcallProc'
	runtime.lua:1253: in function 'runtime.runOpo'
```

**Comment by tomsci** [comment]
Yep, found this one already, will be fixed soon

**Comment by tomsci** [comment]
fixed in #350 (b267e3e03b14b3f2cf77fa673936c1030ecdb056)
